dbt/models/shared_columns.md

```diff
@@ -879,7 +879,7 @@ Parcel has an active homeowner exemption
 
 {% docs shared_column_is_ahsap %}
 Affordable Housing Special Assessment Program indicator. For more information on
-AHSAP, see: https://www.cookcountyassessor.com/affordable-housing
+AHSAP, see: <https://www.cookcountyassessor.com/affordable-housing>
```

Comment:
Maybe a pre-commit thing is doing this. I can remove the arrows and `--no-verify`?

Reply:
I think it's fine to keep the arrows!